ext/pdo/pdo_stmt.stub.php

```diff
@@ -32,10 +32,10 @@ public function errorInfo() {}
     public function execute(?array $params = null) {}
 
     /** @return mixed */
-    public function fetch(int $mode = PDO::FETCH_BOTH, int $cursorOrientation = PDO::FETCH_ORI_NEXT, int $cursorOffset = 0) {}
+    public function fetch(int $mode = 0, int $cursorOrientation = PDO::FETCH_ORI_NEXT, int $cursorOffset = 0) {}
```

Comment:
Alternatives: adding `PDO::FETCH_DEFAULT` or maybe we could make the param nullable

Reply:
I think it makes sense to expose PDO_FETCH_USE_DEFAULT but the problem is that even when you provide 0 to the first argument then it still does not work properly. As a cheap solution, I think 0 is ok.

Reply:
We probably shouldn't make these nullable in a point release for BC reasons (unfortunately, people do extend PDO/PDOStatement). I think using `0` is okay, but exposing `PDO::FETCH_DEFAULT` would probably be cleanest, to avoid having a magic value.

Reply:
Yeah, I'm also happy with `PDO::FETCH_DEFAULT`! For me, `null` would be a tiny bit more correct semantically, but you are right, it's too large of a change for a minor release (I haven't considered inheritance)